tests/test_files.py

```diff
@@ -4,19 +4,16 @@
 TEST_FILE = 'digits1'
 UNICODE_TEST_FILE = 'unicode1'
 
+files_to_test = [TEST_FILE, UNICODE_TEST_FILE]
+
+
 class TestImageFile(unittest.TestCase):
     def test_ground(self):
-        imgf = ImageFile(TEST_FILE)
```

Comment:
please leave these as individual tests - that way, when `test_ground_unicode` stops working, you know what the problem is immediately